src/main/java/com/spotinst/sdkjava/model/bl/azure/elastiGroup/V3/SignalsAzure.java

```diff
@@ -0,0 +1,109 @@
+package com.spotinst.sdkjava.model.bl.azure.elastiGroup.V3;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.spotinst.sdkjava.enums.ElastigroupVmSignalEnumAzure;
+
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+public class SignalsAzure {
+
+    //regionMembers
+    @JsonIgnore
+    private Set<String> isSet;
+    private Integer timeout;
+    private ElastigroupVmSignalEnumAzure type;
+    //end region
+
+    //region Constructors
+    private SignalsAzure(){
+        isSet = new HashSet<>();
+    }
+    //end region
+
+    //region Getters & Setters
+    public Set<String> getIsSet() {
+        return isSet;
+    }
+
+    public void setIsSet(Set<String> isSet) {
+        this.isSet = isSet;
+    }
+
+    public Integer getTimeout() {
+        return timeout;
+    }
+
+    public void setTimeout(Integer timeout) {
+        isSet.add("timeout");
+        this.timeout = timeout;
+    }
+
+    public ElastigroupVmSignalEnumAzure getType() {
+        return type;
+    }
+
+    public void setType(ElastigroupVmSignalEnumAzure type) {
+        isSet.add("type");
+        this.type = type;
+    }
+
+    //region Object overrides
+    @Override
```

Comment:
why do we need override methods in BL class

Reply:
Removed the override methods in BL Class and Tested, call working fine